docker-machine-nfs.sh

```diff
@@ -137,7 +138,7 @@ setPropDefaults()
   prop_mount_options="noacl,async"
   prop_force_configuration_nfs=false
   prop_use_ip_range=false
-  prop_use_ip=
+  prop_timeout=60
```

Comment:
Wouldn't it be better to not set a default value (thus it would be the default value of docker-machine) ?

Reply:
Good point. I updated the PR to not set the timeout value